src/main/java/ratpack/zipkin/internal/RatpackCurrentTraceContext.java

```diff
@@ -23,7 +23,12 @@ public TraceContext get() {
     return registrySupplier.get()
         .maybeGet(TraceContextHolder.class)
         .map(TraceContextHolder::getContext)
-        .orElse(null);
+        .orElseGet(() -> (Execution.isManagedThread())
+          ? Execution.current()
```

Comment:
@llinder I had a question about this from the original PR from @edwardsoo-ss - what's the difference between using `Execution.current()` and `registrySupplier.get()`? `registrySupplier` is just a method ref for `Exection::current` anyway.